downstream/modules/platform/con-aap-upgrades.adoc

```diff
@@ -0,0 +1,15 @@
+
+
+[id="aap-upgrades_{context}"]
+
+= {PlatformNameShort} upgrades
+
+Upgrading to version {PlatformVers} from {PlatformNameShort} 2.1 or later involves downloading the installation package and then performing the following steps:
+
+* Set up your inventory to match your installation environment.
+* Run the {PlatformVers} installation program over your current {PlatformNameShort} installation.
+* Import or generate a new {HubName} API token.
```

Comment:
This is not 'required' when upgrading from 2.1. Although you 'can' still import or generate api token for 2.1 -> 2.2 upgrade, I feel mentioning will cause confusion as most customers probably won't do it. Can you reword (or move section or whatever...) so it's clear this is required for upgrade from 1.2/2.0 only?